api/src/pcapi/core/providers/commands.py

```diff
@@ -4,16 +4,50 @@
 import click
 
 import pcapi.core.providers.repository as providers_repository
-from pcapi.local_providers.provider_manager import synchronize_data_for_provider
-from pcapi.local_providers.provider_manager import synchronize_venue_provider
-from pcapi.local_providers.provider_manager import synchronize_venue_providers_for_provider
+from pcapi.local_providers import provider_manager
 from pcapi.utils.blueprint import Blueprint
 
+from . import models
+from . import tasks
+
 
 blueprint = Blueprint(__name__, __name__)
 logger = logging.getLogger(__name__)
 
 
+def _synchronize_venue_providers_apis() -> None:
+    providers_apis = models.Provider.query.filter(
+        models.Provider.isActive == True, models.Provider.apiUrl != None
+    ).all()
+
+    for provider in providers_apis:
+        venue_provider_ids = [
+            result[0]
+            for result in models.VenueProvider.query.filter_by(providerId=provider.id)
```

Comment:
(détail / nommage) J'aime pas trop beaucoup le nom `result`, ici :) Proposition : 

```suggestion
            id_
            for id_, in models.VenueProvider.query.filter_by(providerId=provider.id)
```